### ISSUE TITLE
TypeError: unsupported operand type(s) for +: 'NoneType' and 'str'

### ISSUE THREAD

**Issue by xilopaint** [Acidham/search-alfred-workflows#4]
```
[06:52:42.605] ERROR: Search Alfred Workflows[Script Filter] Code 1: Traceback (most recent call last):
  File "/Users/<user>/Library/Application Support/Alfred/Alfred.alfredpreferences/workflows/com.apple.alfred.workflow.alfdir/keywords.py", line 14, in <module>
    keyword = keyw + ' ' if withspace else keyw
TypeError: unsupported operand type(s) for +: 'NoneType' and 'str'
```

The exception is raised if the workflow have blank keywords.

**Comment by xilopaint** [comment]
This issue was reintroduced as pointed out here: https://github.com/Acidham/search-alfred-workflows/commit/dc09a2b11d27d7158757551e985288955d045721#r33854543

**Comment by xilopaint** [comment]
@Acidham, I see you change the line again in c2b78878d18103c73a4d683fb5fab98d0ffaaa46. The problem is now I have blank keywords being listed which is noisy and unwanted:

![Screenshot 2019-06-08 at 01 00 59](https://user-images.githubusercontent.com/11052366/59141736-e316f180-8988-11e9-9431-771bac251d06.png)

If you use the code of PR #5 this behaviour will not happen:

![Screenshot 2019-06-08 at 01 03 39](https://user-images.githubusercontent.com/11052366/59141755-41dc6b00-8989-11e9-91f8-9b2a690a4f96.png)

**Comment by Acidham** [comment]
@xilopaint Thx! Argh seems it was too late yesterday. Fixed in 7423bc4
